src/scripts.js

```diff
@@ -129,17 +139,29 @@ function updateButtonText() {
 
 function getRandomUser() {
   currentUser = new User(users[Math.floor(Math.random() * users.length)]);
-  navUserInfo.innerText = currentUser.name;
+  myRecipesTitle.innerText = `What's Cookin' in ${currentUser.name}'s Kitchen?`;
```

Comment:
Great idea tweaking the header and nav text @qrispi - this makes it way clearer to the user that they're looking at their own saved recipes.

Reply:
Shout-out to my beautiful wife for this suggestion! She is the director of my user-experience/quality control dept!

Reply:
This is was a great way to survey and start implementing UX/UI in a meaningful way!